src/main/java/com/ubiquitech/leaveTrack/mvc/HelloController.java

```diff
@@ -29,10 +31,10 @@
 @Controller
 public class HelloController {
       @Autowired
-      private EmployeeServiceImpl employeeService;
```

Comment:
You should try use the interface instead of the implementation class, so that a different implementation could be used without modifying all of the code that depends on it.